exercises/practice/wordy/wordy.wren

```diff
@@ -0,0 +1,10 @@
+//
```

Comment:
I thought we were getting rid of these comment headers in the skeletons.

Reply:
They are still in JS.  Removed.